src/v1/views/mfa-verify/NumberChallengeView.js

```diff
@@ -15,9 +15,11 @@ import hbs from '@okta/handlebars-inline-precompile';
 export default View.extend({
   className: 'number-challenge-view',
   template: hbs`
-      <p data-se="number-challenge-instruction">
-        {{{i18n code="oktaverify.numberchallenge.instruction" bundle="login" arguments="number"}}}
-      </p>
+      {{#if number includeZero=true}}
```

Comment:
https://handlebarsjs.com/guide/builtin-helpers.html#includezero
Show instruction of number is 0